Library/Formula/eg.rb

```diff
@@ -0,0 +1,83 @@
+class Eg < Formula
+  desc "Expert Guide. Norton Guide Reader For GNU/Linux"
+  homepage "http://www.davep.org/norton-guides/"
+  url "http://www.davep.org/norton-guides/eg-1.00.tar.gz"
+  sha256 "e985b2abd160c5f65bea661de800f0a83f0bfbaca54e5cbdc2e738dfbbdb164e"
+
+  depends_on "s-lang"
+
+  patch :DATA
```

Comment:
This is a tool that hasn't been updated in the last 15 years, so before we ping the author, do you have an idea why `clang`/`gcc` doesn't want to compile those escaped newlines in the original source? Granted it's quite a peculiar syntax, but it _should be_ valid C.

```
gcc -Wall -s -O2   -c -o eg.o eg.c
eg.c:177:13: error: expected ')'
            "\
            ^
eg.c:176:11: note: to match this '('
    printf( "Expert Guide Version " EG_VERSION "\n\n"
          ^
eg.c:181:37: error: use of undeclared identifier 'you'
     This program is free software; you can redistribute it and/or modify
                                    ^
eg.c:183:36: error: use of undeclared identifier 'either'
     the Free Software Foundation; either version 2 of the license, or.
                                   ^
eg.c:187:32: error: use of undeclared identifier 'without'
     but WITHOUT ANY WARRANTY; without even the implied warranty of
                               ^
eg.c:192:34: error: expected '(' after 'if'
     along with this program; if not, write to the Free Software
                                 ^
5 errors generated.
make: *** [eg.o] Error 1
```